phase1-coordinator/src/tests.rs

```diff
@@ -1300,6 +1301,386 @@ fn try_lock_blocked_test() -> anyhow::Result<()> {
+    coordinator.add_to_queue(contributor1.clone(), 10)?;
+    coordinator.add_to_queue(contributor2.clone(), 9)?;
+    coordinator.add_to_queue(verifier.clone(), 10)?;
+
+    // Check state in the coordinator.
+    {
+        assert!(coordinator.current_contributor_info(&contributor1).unwrap().is_none());
+        assert!(coordinator.current_contributor_info(&contributor2).unwrap().is_none());
+        assert!(coordinator.current_verifier_info(&verifier).unwrap().is_none());
+    }
+
+    // Check state in the storage.
+    {
+        let coordinator_state = match storage.read().unwrap().get(&Locator::CoordinatorState).unwrap() {
+            Object::CoordinatorState(state) => state,
+            _ => panic!("unexpected object type"),
+        };
+        assert!(
+            coordinator_state
+                .current_contributor_info(&contributor1)
+                .unwrap()
+                .is_none()
+        );
+        assert!(
+            coordinator_state
+                .current_contributor_info(&contributor2)
+                .unwrap()
+                .is_none()
+        );
+        assert!(coordinator_state.current_verifier_info(&verifier).unwrap().is_none());
+    }
+
+    // Update the ceremony to round 1.
+    coordinator.update()?;
+
+    let (contributor1_n_tasks, contributor2_n_tasks) = {
+        let coordinator_state = match storage.read().unwrap().get(&Locator::CoordinatorState).unwrap() {
+            Object::CoordinatorState(state) => state,
+            _ => panic!("unexpected object type"),
+        };
+
+        let contributor1_info = coordinator
+            .current_contributor_info(&contributor1)
+            .unwrap()
+            .expect("expected contributor1 to have info now that it is a current contributor in round 1");
+
+        // Check the coordinator state matches what is in storage.
+        assert_eq!(
+            &contributor1_info,
+            coordinator_state
+                .current_contributor_info(&contributor1)
+                .unwrap()
+                .expect("expected contributor1 to have info now that it is a current contributor in round 1")
+        );
+
+        assert!(!contributor1_info.assigned_tasks().is_empty());
+        assert!(contributor1_info.pending_tasks().is_empty());
+        assert!(contributor1_info.completed_tasks().is_empty());
+        assert!(contributor1_info.disposing_tasks().is_empty());
+        assert!(contributor1_info.disposed_tasks().is_empty());
+
+        let contributor2_info = coordinator
+            .current_contributor_info(&contributor2)
+            .unwrap()
+            .expect("expected contributor2 to have info now that it is a current contributor in round 1");
+
+        // Check the coordinator state matches what is in storage.
+        assert_eq!(
+            &contributor2_info,
+            coordinator_state
+                .current_contributor_info(&contributor2)
+                .unwrap()
```

Comment:
you can probably replace it with `.current_contributor_info(&contributor2)?`

Reply:
Thanks, good tip!